core/rint/src/TRint.cxx

```diff
@@ -146,6 +146,16 @@ TRint::TRint(const char *appClassName, Int_t *argc, char **argv, void *options,
    TApplication(appClassName, argc, argv, options, numOptions),
    fCaughtSignal(-1)
 {
+
+   if (*argc > 1){
+      // Early exit if there are remaining unrecognized options
+      for (auto n = 1; n < *argc; n++){
+         std::cerr << "root: unrecognized option '" << argv[n] << "'\n";
```

Comment:
```suggestion
         Error("TRint", "root: unrecognized option '%s'", argv[n]);
```

Reply:
For our reference, this change leads to the following output:

```
$: root --random -z --nonexistingoption
Error in <TRint::TRint>: root: unrecognized option '--random'
Error in <TRint::TRint>: root: unrecognized option '-z'
Error in <TRint::TRint>: root: unrecognized option '--nonexistingoption'
Error in <TRint::TRint>: Try 'root --help' for more information.
```

I wonder if this is what we want, given that for some other options to `root` there is already the logic of printing to `stderr`: for example `root --help` is [done here](https://github.com/root-project/root/blob/master/core/base/src/TApplication.cxx#L353-L356)

Reply:
Fair enough.  so indeed `std::cerr` or `fprintf(stderr,` makes more sense.